app/scripts/templates/sign_in.mustache

```diff
@@ -72,7 +72,7 @@
         </div>
 
         <div class="button-row">
-          <button id="submit-btn" type="submit" class="disabled">{{buttonSignInText}}</button>
```

Comment:
Instead of removing the `class="disabled"` from the HTML here while this is in an experiment, can `class="disabled"` remain and we add some JS logic to remove the attribute in [afterRender](https://github.com/mozilla/fxa-content-server/blob/78658f094d9862df24aa3f767ef40b24de5d6a61/app/scripts/views/form.js#L94)? Seems like if the experiment doesn't pan out, we are less likely to forget to remove that code than we are to re-add the attributes.